api/v1/dspipeline_types.go

```diff
@@ -82,6 +96,8 @@ type APIServer struct {
 	ToolboxImage string `json:"toolboxImage,omitempty"`
 	// RhelAI image used for ilab tasks in managed pipelines.
 	RHELAIImage string `json:"rhelAIImage,omitempty"`
+	// Enable various pipelines with the deployment of this DSP API server.
```

Comment:
The original wording kind of sounded like it was only on the initial deployment of the DSP API server which it isn't anymore.
```suggestion
	// Enable various managed pipelines on this DSP API server.
```